api/build.gradle

```diff
@@ -865,3 +864,10 @@ dependencies {
     }
   }
 }
+
+compileGeneratedKotlin {
```

Comment:
are these simply to make implicit dependencies explicit, or is there something else going on here?

Reply:
cool, good change then!

Reply:
should this be named "implementation" now?

Reply:
[mob chat] the gradle warning specified this name, so there must be some implicit "compile" naming happening